core/controllers/base.py

```diff
@@ -300,21 +300,7 @@ def render_downloadable_file(self, values, filename, content_type):
             'attachment; filename=%s' % filename)
         self.response.write(values)
 
-    def _get_logout_url(self, redirect_url_on_logout):
-        """Prepares and returns logout url which will be handled
-        by LogoutPage handler.
-
-        Args:
-            redirect_url_on_logout: str. URL to redirect to on logout.
-
-        Returns:
-            str. Logout URL to be handled by LogoutPage handler.
-        """
-        return current_user_services.create_logout_url(redirect_url_on_logout)
```

Comment:
Should the create_logout_url() function be deleted from current_user_services? Is it being used anywhere else?

Reply:
It is not being used anywhere. So I deleted it.